internal/constants/scripts.go

```diff
@@ -29,12 +29,13 @@ package constants
 const PreNetworkConfigScript = `#!/bin/bash
 
 # The directory that contains nmconnection files of all nodes
-NMCONNECTIONS_DIR=/etc/assisted/network
+SRC_DIR=/etc/assisted/network
+DST_DIR=/etc/coreos-firstboot-network
```

Comment:
So is this now the destination regardless of if we're running during initrd or from ignition?

Please make sure you test both full and minimal ISO with this change.

Reply:
No @carbonin , it isn't the same destination, unfortunately. I was trying to make this work only for the minimal ISO configuration. Now that it works I have adjusted it so that it works for both the minimal and full ISO. Will be in the next iteration of the patch.